verification/io/router.gobra

```diff
@@ -23,33 +23,33 @@ package io
 // This function corresponds to function 'core' in the IO spec
 ghost
 decreases
-pure func core_as_set() set[IO_as]
+pure func (dp DataPlaneSpec) core_as_set() set[IO_as] //TODO: how
```

Comment:
Can you clarify?

Reply:
Sorry, this was just a reminder for me and needs to be dropped. This function is concretized in PR (#231).